ffwd-client/pom.xml

```diff
@@ -14,6 +14,15 @@
     <description>FastForward client for Java.</description>
 
     <dependencies>
+        <dependency>
+            <groupId>com.google.auto.value</groupId>
```

Comment:
Can we avoid using auto value and try keep this client lean?

Reply:
Same as above